config/initializers/defra_ruby_exporters.rb

```diff
@@ -0,0 +1,14 @@
+module DefraRuby
+  module Exporters
+    BATCH_SIZE = (ENV["EXPORT_SERVICE_BATCH_SIZE"] || 3000).to_i.freeze
+
+    AWS_REGION = (ENV["AWS_REGION"] || "eu-west-1").freeze
+
+    EPR_EXPORT_AWS_CREDENTIALS = Aws::Credentials.new(
+      (ENV["AWS_DAILY_EXPORT_ACCESS_KEY_ID"] || raise("Environment variable AWS_DAILY_EXPORT_ACCESS_KEY_ID has not been set")),
```

Comment:
Nice 😁 👍 . Probably a pattern we should be using for a few other values which don't have defaults in the app.

Reply:
👍